File: SilverdawnSoftware.Invoice.Actions/Views/Models/AddressView.cs
// ALLOWOVERWRITE-F719A1DDDB275F7BACD4BD058114A6B8


using System;
using System.Collections.Generic;
using System.Linq;
using System.Runtime.Serialization;


namespace SilverdawnSoftware.Invoice.CQRS.Querys.Models.Models 
{
	[DataContract]
    public partial class AddressView : CQRSBase
    {    
		[DataMember]
		public int AddressId  { get; set; } 
		
		[DataMember]
		public string AddressLine1  { get; set; } 
		
		[DataMember]
		public string AddresLine2  { get; set; } 
		
		[DataMember]
		public string City  { get; set; } 
		
		[DataMember]
		public string PostZipCode  { get; set; } 
		
		[DataMember]
		public string StateCounty  { get; set; } 
		
		[DataMember]
		public string Country  { get; set; } 
		
		[DataMember]
		public string AddressExternalRef  { get; set; } 
		
		[DataMember]
		public string AddressLine3  { get; set; } 
		
    	
   	
    	 public AddressView()
        {
            this.__CQRSCommandName = "SilverdawnSoftware.Invoice.CQRS.Querys.Models.AddressView";
        }
    }
}

File: CRUDAccess/InvoiceWebClient/src/app/modules/admin/Address/Models/AddressView.ts
// ALLOWOVERWRITE-1816013EDA9AAEFE76EDEFBEADDFE59D


    export interface IAddressView {
      addresLine2: string;    	
      addressExternalRef: string;    	
      addressId: number;    	
      addressLine1: string;    	
      addressLine3: string;    	
      city: string;    	
      country: string;    	
      postZipCode: string;    	
      stateCounty: string;    	

					    customerCustomerId : number;   
					    customerName : string;   
    }
    
    export class AddressView implements IAddressView {
      addresLine2: string;    	
      addressExternalRef: string;    	
      addressId: number;    	
      addressLine1: string;    	
      addressLine3: string;    	
      city: string;    	
      country: string;    	
      postZipCode: string;    	
      stateCounty: string;    	

					    customerCustomerId : number ;   
					    customerName : string;   


	  constructor() {
	  }
    }

